Cargo.toml

```diff
@@ -112,16 +112,22 @@ pre-release-replacements = [
 [badges]
 codecov = { repository = "clap-rs/clap-verbosity-flag" }
 
+[features]
+serde = ["dep:serde", "log/serde"]
+
 [dependencies]
 log = "0.4.1"
 clap = { version = "4.0.0", default-features = false, features = ["std", "derive"] }
+serde = { version = "1.0", optional = true, features = ["derive"] }
```

Comment:
Did I miss seeing a `derive` being used?

Reply:
No, I included this and then realized this required manual serialization. Removing.

Reply:
With the simplification in 2dda2ab to use the `From` implemenation instead of a custom implementation, I added the derive feature back in.